tests/utils/command_line_params.cpp

```diff
@@ -788,6 +791,7 @@ void add_eigen_option_group(std::shared_ptr<QUDAApp> quda_app)
   opgroup->add_option("--eig-ortho-block-size", eig_ortho_block_size,
                       "The block size to use when orthonormalising vectors in hybrid modified Gram-Schmidt"
                       "0 for always Classical, 1 for Modified, n > 1 for Hybrid)");
+  opgroup->add_option("--eig-eval-block-size", eig_eval_block_size, "The block size used when computing eigenvalues in the eigensolver");
```

Comment:
`block size` -> `batching size`? This connects to my comment on documenting the `eval_block_size` variable earlier.

Reply:
https://github.com/lattice/quda/commit/fc0762d6152ff9b3f61b4c5d1787273c3d64ca02